.prettierignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+dist
+package-lock.json
+libs
```

Comment:
Maybe worth duplicating excludes from .gitignore?
```
.grunt
node_modules
tmp
bin
obj
dist
.idea
.eslintcache
```

Reply:
I tidied up the `gitignore` as many of these aren't valid any more and added the last 2 to prettierignore.